### PR TITLE
Add a new dialogue to Supreme Bao

### PR DESCRIPTION
**Pull request by setbreath**
Please review this, @casey-collab

### CODE REVIEW

File: Chinese/PanicStation/panic.md [hunk 1]
```diff
@@ -0,0 +1 @@
+至尊宝：其实我就是一条狗 一条害怕😢的狗
```
Comment (casey-collab) [inline Chinese/PanicStation/panic.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by setbreath** [comment]
Link added. Please check again. 
Thanks!

**Comment by setbreath** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :cake:

**Pull request merged**